profiles/drop.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1.0
+name: drop
```

Comment:
Does it make sense to name it "drop-mon"  or something similar that indicates that's a monitor-kind of profile?
I'm thinking that the name "drop" might make someone think that's just adding skb-drop and might combine it with lots of probes in, say, the TCP stack. The result would be quite noisy with lots of stack traces.